CONTRIBUTING.md

```diff
@@ -0,0 +1,41 @@
+# Contributing to DBOS-TS
+
+Thank you for considering contributing to DBOS-TS. We welcome contributions from everyone, whether it's bug fixes, feature enhancements, documentation improvements, or any other form of contribution.
```

Comment:
```suggestion
Thank you for considering contributing to DBOS Typescript SDK. We welcome contributions from everyone, whether it's bug fixes, feature enhancements, documentation improvements, or any other form of contribution.
```

Reply:
I think we can either call it `dbos-ts` or use the full name `DBOS Typescript SDK`.